cmake/modules/SwiftSetIfArchBitness.cmake

```diff
@@ -6,7 +6,9 @@ function(set_if_arch_bitness var_name)
       "" # multi-value args
       ${ARGN})
 
-  if("${SIA_ARCH}" STREQUAL "i386" OR
+  if("${SIA_ARCH}" STREQUAL "avr")
+      set("${var_name}" "${SIA_CASE_16_BIT}" PARENT_SCOPE)
```

Comment:
This probably also needs an extension on line 5, otherwise I think it's not doing anything?

Reply:
I'll move this change to the 16 bit PR because I think this only affects compiling the standard library? In which case it's further down the chain of PRs we are planning before this becomes important.